pinot-query-runtime/src/main/java/org/apache/pinot/query/runtime/operator/MailboxSendOperator.java

```diff
@@ -120,7 +120,7 @@ public List<MultiStageOperator> getChildOperators() {
   @Override
   public String toExplainString() {
     _dataTableBlockBaseOperator.toExplainString();
-    LOGGER.debug(_operatorStats.toString());
+    LOGGER.info(_operatorStats.toString());
```

Comment:
can we move these to `MultiStageOperator` looks like they are all identical on every concrete impl

Reply:
we can also move `_operatorStats.startTimer();` and `_operatorStats.endTimer()` to `MultiStageOperator` and overwrite the `nextBlock()` method.

Reply:
Can move in next PR since it is urgent.